picamera2/converters.py

```diff
@@ -1,9 +1,13 @@
 import numpy as np
 
 
-YUV2RGB_JPEG      = np.array([[1.0,   1.0,   1.0  ], [0.0, -0.344, 1.772], [1.402, -0.714, 0.0]])
-YUV2RGB_SMPTE170M = np.array([[1.164, 1.164, 1.164], [0.0, -0.392, 2.017], [1.596, -0.813, 0.0]])
-YUV2RGB_REC709    = np.array([[1.164, 1.164, 1.164], [0.0, -0.213, 2.112], [1.793, -0.533, 0.0]])
+YUV2RGB_JPEG = np.array([[1.0, 1.0, 1.0], [0.0, -0.344, 1.772], [1.402, -0.714, 0.0]])
+YUV2RGB_SMPTE170M = np.array(
+    [[1.164, 1.164, 1.164], [0.0, -0.392, 2.017], [1.596, -0.813, 0.0]]
+)
+YUV2RGB_REC709 = np.array(
+    [[1.164, 1.164, 1.164], [0.0, -0.213, 2.112], [1.793, -0.533, 0.0]]
+)
 
```

Comment:
I think I prefer the original formatting.  Or possibly all 3 matrices to look  something like:

```
YUV2RGB_REC709 = np.array(
    [ [1.164, 1.164, 1.164],
      [0.0, -0.213, 2.112],
      [1.793, -0.533, 0.0] ]
)
```